.circleci/config.yml

```
@@ -74,9 +74,11 @@ jobs:
       # see https://circleci.com/orbs/registry/orb/adobe/helix-post-deploy
       # for more available parameters
       - helix-post-deploy/monitoring:
+          targets: aws, google
```

```suggestion
          targets: aws
```

(content bus it not deployed on google yet)